signatures_test.go

```diff
@@ -1406,15 +1407,19 @@ func TestMultipleSignatures(t *testing.T) {
 		SetVerifyCreated(false).SetKeyID("test-key-ecc-p256"), Headers("@method", "@authority", "@path", "content-digest",
 		"content-type", "content-length"))
 	assert.NoError(t, err, "cannot create verifier1")
-	_, err = verifyRequestDebug("sig1", *verifier1, req)
```

Comment:
Please add new tests rather than changing existing ones, since we're not deprecating the existing library's API.